messenger.go

```diff
@@ -305,6 +305,16 @@ func (m *Messenger) SendWithReplies(to Recipient, message string, replies []Quic
 	return response.TextWithReplies(message, replies)
 }
 
+// Attachment send image, audio, video and file attachment.
+func (m *Messenger) Attachment(to Recipient, dataType, url string) error {
```

Comment:
When the constant attachment type below is implemented, this signature will need to change too.